### DISCUSSION TITLE
Make badges independent of forked repositories

### DISCUSSION THREAD

**Discussion by dorosch** [community/community#120524] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello,

First of all, I want to say thank you for developing the platform and community.

My proposal is to make profile badges independent of the forked repositories for contributions to which these badges appeared. I deleted the repository from my list and along with it the icon from my profile disappeared.

**Comment by Gajarthan** [comment]
Hello dorosch,

Thank you for sharing your appreciation for the platform and community! It sounds like your issue revolves around how profile badges tied to contributions in forked repositories are handled. Currently, if you delete a forked repository, any badges linked to contributions in that repository may disappear from your profile as well. This can be frustrating if you're trying to clean up your repository list but still want to display your contributions.

I understand the importance of having a more flexible system where badges remain on your profile regardless of the status of the forked repositories. This would help in acknowledging and showcasing your contributions permanently, even if the associated repository no longer exists on your profile.

It might be beneficial to suggest this feedback directly to GitHub through their official feedback channels or on the GitHub Community Discussions. This allows your proposal to reach the right people who can consider such improvements.

In the meantime, if you have any further queries or need assistance with another GitHub-related topic, feel free to ask!

**Reply by ettaboyle** [reply]
@Gajarthan Thank you for your contributions to helping other users in the GitHub Community! Its great to see your comments on posts. Please take note, the tone and the phrasing you use in this response gives the impression that you are a GitHub Staff member. 

Impersonating GitHub staff violates our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md). We ask that members contribute in a positive and constructive way. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by ettaboyle** [comment]
Thank you for this feedback, @dorosch

**Comment by Hazrat-Ali9** [comment]
Thank You For this Feedback.
